crates/sui-core/src/authority/transaction_deferral.rs

```diff
@@ -54,7 +54,7 @@ impl DeferralKey {
                 deferred_from_round: 0,
             },
             Self::ConsensusRound {
-                future_round: consensus_round.checked_add(1).unwrap(),
+                future_round: consensus_round.saturating_add(1),
```

Comment:
If an epoch runs for long enough that `consensus_round` reaches `u64::MAX` (or gets this close to it), then I think we want to panic, because there is no safe way for the system to handle that

Keep in mind that at our current rate of 14 rounds/s it would take an epoch of over 4e10 years for us to reach this limit, even if we somehow 1000x this rate (impossible I think) then it would take 10s of millions of years :)

Reply:
Additionally, if we did reach this point, doing a saturating add would cause incorrect behavior.